src/bin/lpython.cpp

```diff
@@ -1739,6 +1841,14 @@ int main(int argc, char *argv[])
             return 1;
         }
 
+        if (to_jit) {
+            compiler_options.emit_debug_info = false;
+            compiler_options.emit_debug_line_column = false;
+            compiler_options.generate_object_code = false;
+            return execute_python_using_jit(arg_file, runtime_library_dir,
+                    lpython_pass_manager, compiler_options, time_report);
+        }
```

Comment:
> One of them says error: use of undeclared identifier 'execute_python_using_jit'. Even though it is declared and defined in the preceding lines.

This should be wrapped inside `#ifdef HAVE_LFORTRAN_LLVM` `#endif`. Hopefully, it should fix the CI failure.